### ISSUE TITLE
how can I train with conditional is True?

### ISSUE THREAD

**Issue by xinyaojiejie** [huangzh13/StyleGAN.pytorch#15]
Could you give an example for how to train a model under conditional is set as True? Or give some suggestion on it ? Thx!

**Comment by huangzh13** [comment]
Implemented in commit `64f1bad` and pushed to `master`.

This adds a complete conditional-training example:

- documents the expected `ImageFolder` layout and class-label mapping;
- updates `configs/sample_conditional.yaml` with a portable example;
- validates that `n_classes` matches the number of class directories before training;
- passes labels through the training entry point to the conditional generator and discriminator;
- adds dataset, conditional forward, and one-step training smoke tests.

Run it with:

```shell
python train.py --config configs/sample_conditional.yaml
```

The local environment did not have `torch`/`pytest`, so the full test suite could not be run locally; syntax and diff checks passed.
